### PR TITLE
ip subscription manager

### PR DESCRIPTION
**Pull request by coot**

### CODE REVIEW

File: stack.yaml [hunk 1]
```diff
@@ -39,22 +39,22 @@ extra-deps:
   - tasty-hedgehog-1.0.0.1
     # iohk-monitoring-framework currently not pinned to a release
   - git: https://github.com/input-output-hk/iohk-monitoring-framework
-    commit: e67ac43fc26e3a47ffcf1a10666f2ccd34ce8b1a
+    commit: 3ac2191fc4c68b46b7f6c08e2873f71949cc68e7
```
Comment (CodiePP) [inline stack.yaml]: please use commit: b2022cf7f5925e486f9e0a31f6f6d02145d5ebda

File: nix/.stack.nix/iohk-monitoring.nix [hunk 2]
```diff
@@ -155,8 +155,8 @@
     } // {
     src = (pkgs.lib).mkDefault (pkgs.fetchgit {
       url = "https://github.com/input-output-hk/iohk-monitoring-framework";
-      rev = "e67ac43fc26e3a47ffcf1a10666f2ccd34ce8b1a";
-      sha256 = "0xwaclxvh37hbp4xj9vlsn0ir7d11jlyl6pnnpi6h1nwggqrzm85";
+      rev = "3ac2191fc4c68b46b7f6c08e2873f71949cc68e7";
```
Comment (CodiePP) [inline nix/.stack.nix/iohk-monitoring.nix]: please use commit: b2022cf7f5925e486f9e0a31f6f6d02145d5ebda

File: stack.yaml (hunk 1 above)
Comment (coot) [reply stack.yaml]: Done

**Comment by coot** [comment]
It was a draft because it used bring things from ahead of `master` of `ouroboros-network`, since that's already merged we can merge it when the CI passes.

**Pull request merged**